etc/scripts/docenizer.py

```diff
@@ -56,39 +89,62 @@ def instr_name(i):
         return match.group(1)
 
 
-def get_description(section):
+def get_section_description_pars(section):
+    l = []
     for sub in section:
-        descr = sub.get_text().strip()
+        l.append(sub.get_text())
+    return l
+
+
+def get_strong_description_pars(strong):
+    sibling = strong.parent
+    l = []
+    while True:
```

Comment:
Do you think this could be changed to  `while hasattr(sibiling, 'next_sibiling'):` and remove the break if?

Reply:
This would not work in the very special case where `hasattr(sibling, 'p')` is true, but `hasattr(sibling, 'next_sibling')` is false.